u-root.go

```diff
@@ -98,7 +102,15 @@ func Main() error {
 	// Currently allowed formats:
 	//   Go package imports; e.g. github.com/u-root/u-root/cmds/ls
 	//   Paths to Go package directories; e.g. $GOPATH/src/github.com/u-root/u-root/cmds/*
-	pkgs := flag.Args()
+	var pkgs []string
+	for _, a := range flag.Args() {
+		p, ok := templates[a]
+		if !ok {
+			pkgs = append(pkgs, a)
+			continue
+		}
+		pkgs = append(pkgs, p...)
+	}
```

Comment:
we also need to remove duplicates from overlapping templates, so maybe a `map[string]string` - if it's not done later already